PageSpeed_Api.php

```diff
@@ -1,128 +1,544 @@
 <?php
-namespace W3TC;
-
 /**
- * Google Page Speed API
+ * File: PageSpeed_Api.php
+ *
+ * @package W3TC
  */
-define( 'W3TC_PAGESPEED_API_URL', 'https://www.googleapis.com/pagespeedonline/v5/runPagespeed' );
+
+namespace W3TC;
 
 /**
- * class PageSpeed_Api
+ * PageSpeed API.
  */
 class PageSpeed_Api {
 	/**
-	 * API Key
+	 * Config
+	 *
```

Comment:
Updated all files/classes/methods to show an @since of 2.3.0 due to the fact that the entire feature will have been overhauled. See https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#since-section-changelogs